examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<ol><li>Too complicated, write sth like "To work with the reconciliators in pipeline you need to use <code>HierarchicalPipeline</code>"</li><li>May be it is better to write about the reconciliation methods and there parameters and then show how to use them with and without pipeline</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<ol><li>This proposal does not reflect how pipeline uses reconciliation.</li><li>Right now, all the information about each method is packed together in sections, making it easier to locate the necessary parts.</li></ol>

Reply:
<p>Ok, then let's slightly rework:</p><p><code>HierarchicalPipeline</code>&nbsp;provides the ability to perform forecasts reconciliation in pipeline. There is a couple if important statements:t</p><ol><li>Transforms and model work with the dataset on the <strong>source</strong> level. </li><li>Forecasts are automatically reconciliated to the <strong>target</strong> level, metrics reported for <strong>target</strong> level as well.</li></ol>